src/book/04-guides/01-nubots/01-getting-started.mdx

```diff
@@ -8,14 +8,161 @@ slug: /guides/nubots/getting-started
 
 ## Prerequisites
 
+A Linux operating system is recommended for running NUbots code, but any operating system should work. Currently Windows is unsupported but this may change in the future. You can find free downloads for [Ubuntu](https://ubuntu.com/) online. Instructions for getting started with NUbots code can also be found at [the NUbots repository](https://github.com/NUbots/NUbots).
```

Comment:
We should also choose between NUbook and the NUbots readme for NUbots getting started documentation, instead of maintaining both with the same content. If we go with NUbook, then a basic introduction in the README with a link to this page will suffice. Then NUbook won't have to link back.

Reply:
NUbook should have it, I can get rid of that sentence then once this is merged in update the NUbots README.